Flashcards.ngalantino/Flashcards.ngalantino/StudySession.cs

```diff
@@ -0,0 +1,5 @@
+public class StudySession {
+    public string Stack { get; set; }
```

Comment:
string in c# is a non-nullable type, so try to make the string property nullable or initialize the value.
"public string? Stack {get; set;}" or
"public string Stack {get; set;} = '' "